database/migrations/2024_03_01_095414_create_archived_statements_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('archived_statements', function (Blueprint $table) {
+            $table->id();
+            $table->integer('platform_id');
+            $table->string('puid', 500);
+            $table->string('uuid', 36)->nullable();
+            $table->timestamp('date_received')->nullable();
+            $table->timestamps();
+
+            $table->unique(['platform_id', 'puid'], 'platform_puid_unique');
```

Comment:
let's just go with index, if do unique we recreate the problem but in another table.

Reply:
yes !